### PR TITLE
isNaN and isFinite added

### PR DESCRIPTION
**Pull request by kshirish**

### CODE REVIEW

**Comment by stoeffel** [comment]
Would be cool if you would link to the docs in mdn i.e. https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/isNaN
see [bind](https://github.com/1-liners/1-liners/blob/master/documentation/README.md#bind) as an example. otherwise this looks good. thx for the pr.

**Comment by kshirish** [comment]
Did that just now.

File: module/isFinite.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/**
+ * @module 1-liners/isFinite
+ * 
+ * @description
+ *
+ * Check if the number is finite
+ * Same as https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isFinite
```
Comment (stoeffel) [inline module/isFinite.js]: > Same as [`isFinite(testValue)`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/isFinite)

Would be nicer.

File: module/isNaN.js [hunk 2]
```diff
@@ -0,0 +1,16 @@
+/**
+ * @module 1-liners/isNaN
+ * 
+ * @description
+ *
+ * Check if the number is NaN
+ * Same as https://developer.mozilla.org/en/docs/Web/JavaScript/Reference/Global_Objects/isNaN
```
Comment (stoeffel) [inline module/isNaN.js]: Same here

**Comment by davidchase** [comment]
@stoeffel @kshirish, wouldnt this PR effectively overwrite the browser globals `isNaN` and `isFinite` ?
also it seems like es6 provides new better checks such as [Number.isNaN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isNaN) and [Number.isFinite](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isFinite) without any type coercing.

thoughts?

**Comment by tomekwi** [comment]
I normally do `const {isNaN} = Number` for these things anyway. So @davidchase :+1:

**Comment by stoeffel** [comment]
Aggreed with you both lets close this then.

**Pull request closed without merging**